bids/modeling/model_spec.py

```diff
@@ -7,27 +7,6 @@
 from bids.utils import convert_JSON
 
 
-def create_model_spec(df, model, *args, **kwargs):
```

Comment:
This function is only used by `BIDSStatsModelsNodeOutput` so I found it just confuse the logic to keep it outside the main flow of logic in `BIDSStatsModelsNodeOutput.__init__`